templates/nextjs/fuels.config.ts

```diff
@@ -13,4 +13,6 @@ export default createConfig({
   output: './src/sway-api',
   fuelCorePort,
   providerUrl: NODE_URL,
+  forcPath: 'fuels-forc',
+  fuelCorePath: 'fuels-core',
```

Comment:
@petertonysmith94 We probably don't want it to go like this to users.

Template-wise, we should omit `forcPath` and `fuelCorePath` entirely.

After all, we require users to have `fuelup` + `forc` + `fuel-core` installed.

cc @Dhaiwat10

Reply:
Agreed

Reply:
We did this manual testing today, and it was essential to spot some bugs.

We should look into automating this in CI.

- https://github.com/FuelLabs/fuels-ts/issues/2170

Reply:
Thanks!